CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/Extensions.swift

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
What is this file for?

Reply:
@sridevi-15 might know more details

Reply:
this is to convert replicator status api  and activity level api  from enum to string.